gen-ai/orchestrator-server/src/main/python/tock-llm-indexing-tools/index_documents.py

```diff
@@ -164,6 +163,18 @@ def generate_ids_for_each_chunks(
     return splitted_docs
 
 
+def add_title_to_text(
+    splitted_docs: Iterable[Document],
+) -> Iterable[Document]:
+    """Add 'title' from metadata to Document's page_content for better semantic search."""
+    for doc in splitted_docs:
+        # Add title to page_content
+        if 'title' in doc.metadata:
+            title = doc.metadata['title']
+            doc.page_content = f'{title}\n\n{doc.page_content}'
```

Comment:
Can you add a comment indicating that this pattern is used in rag_chain.py so that if someone change it here they will know that it needs to be updated also on the chain script.